addons/event/data/event_demo.xml

```diff
@@ -218,7 +218,7 @@
     </record>
 
     <record id="event.event_7" model="event.event">
-        <field name="name">OpenWood Collection Online Reveal</field>
```

Comment:
By the way there is a test in test_event_full using the same naming (just to be able to run a tour automatically or locally). Is it still matching ?

Reply:
Did not find such a test, since runbot tests are all green, this means that it's still working right?

Reply:
If you look in test_event_full/static/src/js/tours/wevent_register_tour.js and test_event_full/tests/common.py we have some trick to allow running the tour both automatically and on demo data. If you update this demo you should also update those files to ensure we can always run this tour. It is used mainly when working on event sub modules, ensuring we can still run the global tour.

Reply:
I can see that now, thanks for the hint, If kept unchanged the tour will indeed not work properly.

It has now hopefully been corrected, you can take a look again!